### DISCUSSION TITLE
reCaptcha v3 is failing when executed from Cypress managed browser

### DISCUSSION THREAD

**Discussion by mturza** [cypress-io/cypress#30010] [Questions and Help]
Hello.
We are experiencing a problem when calling recaptcha in our tests. 
When performing the same actions manually, no problem is present.
If we run the test in Cypress, we are getting `error("Error: Invalid API parameter(s). Try reloading the page.")`.
The problem is the same as here https://github.com/cypress-io/cypress/discussions/24128
Unfortunately, there is no explanation of what is causing the error.

Note: using Cypress 13.13.1

**Comment by rafaelflima1910** [comment]
I'm facing the same problem. I've done several things, and I can't find a solution. I've tested it with other scenarios and it worked, but now the problem only occurs with a specific scenario.
